src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -150,106 +149,129 @@ Dashboards
+* [Drop data](/docs/accounts/accounts/data-management/drop-data-using-nerdgraph)
+      </td>
     </tr>
 
     <tr>
       <td>
-Manage keys      </td>
+Manage keys      
+      </td>
 
       <td>
 [Create and manage keys](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys) (license keys used for data ingest, and user keys)
       </td>
     </tr>
 
+
+
+
+
     <tr>
       <td>
-Manage data      </td>
+New Relic One apps  </td>
 
       <td>
-* [Convert event data to metric data](/docs/telemetry-data-platform/ingest-manage-data/convert-data-metrics/analyze-monitor-data-trends-metrics)
-* [Drop data](/docs/accounts/accounts/data-management/drop-data-using-nerdgraph)
+[Build a New Relic One app](https://developer.newrelic.com/explore-docs/query-and-store-data)
       </td>
     </tr>
 
     <tr>
       <td>
-Distributed tracing      </td>
+Partners and resellers</td>
 
       <td>
-* [Query distributed tracing data](/docs/apis/nerdgraph/examples/nerdgraph-distributed-trace-data-tutorial)
-* [Configure Infinite Tracing](/docs/apis/nerdgraph/examples/configure-infinite-tracing-graphql) 
-      </td>
+[Manage subscriptions](/docs/apis/nerdgraph/tutorials/provisions-your-subscriptions-nerdgraph) (only for partners using [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-plans))      </td>
     </tr>
 
     <tr>
       <td>
-New Relic One apps  </td>
+Querying data
+      </td>
 
       <td>
-[Build a New Relic One app](https://developer.newrelic.com/explore-docs/query-and-store-data)
+[Query using NRQL](/docs/apis/nerdgraph/examples/nerdgraph-nrql-tutorial) (our query language)
       </td>
     </tr>
 
     <tr>
       <td>
-Cloud integrations (AWS, Azure, GCP)  </td>
-
+Service levels      
+      </td>
+   
       <td>
-[Configure cloud integrations](/docs/apis/nerdgraph/examples/nerdgraph-cloud-integrations-api-tutorial)
+[Configure and manage service levels](/docs/apis/nerdgraph/tutorials/nerdgraph-slm)      
       </td>
     </tr>
 
     <tr>
       <td>
-Partners and resellers</td>
+Tags      
+      </td>
 
       <td>
-[Manage subscriptions](/docs/apis/nerdgraph/tutorials/provisions-your-subscriptions-nerdgraph) (only for partners using [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-plans))      </td>
+[Add and manage tags](/docs/apis/nerdgraph/examples/nerdgraph-tagging-api-tutorial)
+      </td>
     </tr>
 
- <tr>
+    <tr>
       <td>
-Data partitions</td>
+Workloads      
+      </td>
 
       <td>
-[Manage data partitions](/docs/apis/nerdgraph/examples/nerdgraph-data-partition-rules-tutorial)</td>
+[View and configure workloads](/docs/apis/nerdgraph/tutorials/nerdgraph-workloads-api-tutorials)      
+      </td>
     </tr>
 
-<tr>
+    <tr>
       <td>
-Date retention</td>
+Your monitored entities
```

Comment:
Maybe just "Entities".  I wouldn't think to scan to Your to find stuff about entities.